entries/t/tpg.com.au.json

```diff
@@ -0,0 +1,15 @@
+{
+  "TPG": {
+    "domain": "tpg.com.au",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://support.tpg.com.au/multi-factor-authentication-tpg",
```

Comment:
Could you take a screenshot? They seem to have geo-blocked me 😅

Reply:
We reviewed at the same time again!

Reply:
![image](https://github.com/2factorauth/twofactorauth/assets/7688583/df282129-ba5f-44e5-9ccd-1c737310a53a)